pandas/core/frame.py

```diff
@@ -4955,9 +4955,9 @@ def pivot(self, index=None, columns=None, values=None):
             Column to use to make new frame's index. If None, uses
             existing index.
         columns : string or object
-            Column to use to make new frame's columns.
-        values : string or object, optional
-            Column to use for populating new frame's values. If not
+            Column name to use to make new frame's columns
+        values : string, object or a list (0.23.0) of the previous, optional
```

Comment:
Instead of mentioning the `(0.23.0)` here, can you instead add a 

```
.. versionchanged:: 0.23.0
    Also accept list of column names.
```

to the parameter description?